README.md

```diff
@@ -82,9 +82,39 @@ System landscape view where Valkyrie fits in:
 Components view of Valkyrie, how the internals connect to each other.
 ![components view](/structurizr/structurizr-1-Valkyrie-Components.png)
 
+### Codebase structure
```

Comment:
Open question: should we create separate markdown files for this and link to from README or do we keep it here? Seems like two different schools when I look at other projects, some keep very slim READMEs and others keep all their docs in there 😄

Reply:
My opinion is to keep it in one, or as few as possible. I want to jump around as little as possible. Keep as much information as possible in one place so the reader don't have to keep track of where a piece of information they read earlier is. 
My preference at least, also from experience of reading other docs. I definitely prefer those where all is kept in one. At least when it comes to README/markdown docs in a repo. A doc website usually have more navigation help etc.